core/src/main/java/io/kestra/core/plugins/PluginIdentifier.java

```diff
@@ -1,7 +1,24 @@
 package io.kestra.core.plugins;
 
+import org.apache.commons.lang3.tuple.Pair;
 
 /**
  * Represents the fully qualify identifier of a Kestra's plugin.
  */
-public interface PluginIdentifier { }
+public interface PluginIdentifier {
+
+    /**
+     * Helper method for parsing a string plugin identifier to extract a type and version.
+     *
+     * @param identifier    a string type identifier.
+     * @return  a {@link Pair} of (type, version).
+     */
+    static Pair<String, String> parseStringIdentifier(final String identifier) {
+        int index = identifier.indexOf(":");
```

Comment:
```suggestion
        int index = identifier.indexOf(':');
```

Slighly more performant